### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by JuanBohorquezPerficient**

### CODE REVIEW

File: cypress/page/download.page.ts
Comment (holgiosalos) [inline cypress/page/download.page.ts]: is this needed?

File: cypress/page/download.page.ts
Comment (holgiosalos) [inline cypress/page/download.page.ts]: are you sure that the wait is needed?

Comment (JuanBohorquezPerficient) [reply cypress/page/download.page.ts]: Oh sorry, I forgot to delete those lines
 I'm in my way to erase that stuff
thx, Holmes

File: cypress/e2e/iframe.cy.ts
Comment (holgiosalos) [inline cypress/e2e/iframe.cy.ts]: as this is for the next step, could you deleted and wait until merge the changes related to upload-download page

Comment (JuanBohorquezPerficient) [reply cypress/e2e/iframe.cy.ts]: Ok, I've deleted all the stuff from the next step, thx

**Review by holgiosalos — APPROVED** [review]
For next PR you can change the branch, it does not have to be always the setup branch. It is only a suggestion. You can continue using it, but it is kindof weird see a lot of commits in each PR

**Pull request merged**
